README.md

```diff
@@ -85,14 +85,16 @@ If you are new to the group, check out our [New Members Page](NEWMEMBERS.md) and
 
 ## Members
 
-### SIG Chairs
+### SIG Chairs Emiritus
```

Comment:
spelling:  Emeritus

Nit: would expect Emeritus positions to be at the bottom, since likely to be less active than current chairs & tech leads, but ok by me if you want to highlight the awesome Dan Shaw this way :)